src/validators/function.rs

```diff
@@ -46,6 +46,14 @@ macro_rules! impl_build {
                     function_name(function)?,
                     validator.get_name()
                 );
+                let config_hide_input = match config {
+                    Some(c) => c.get_item("hide_input_in_errors"),
+                    None => None,
+                };
+                let hide_input = match config_hide_input {
+                    Some(t) => t.is_true().unwrap_or(false),
+                    None => false,
+                };
```

Comment:
```suggestion
                let hide_input_in_errors: bool = config.get_as(intern!(py, "hide_input_in_errors")).unwrap_or(false);
```
we have some nice utilities for this stuff 😄